.github/workflows/dotnet-ci.yml

```diff
@@ -7,10 +7,7 @@ name: dotnet-ci
 on:
   workflow_dispatch:
   push:
-    branches: [ "main" ]
-    paths:
-      - "dotnet/**"
```

Comment:
I like the path filters, it'd be nice to just include `.github/workflows/` as a path.